index.html

```diff
@@ -10,31 +10,57 @@
 </head>
 <body>
     <header>
-        <nav>
+        <nav class="nav-menu">
            <ul>
-              <li><a href="#"> <img src="" alt="apple icon"></a></li>     
+              <li><a href="#"> <i class="fa fa-apple"></i></a></li>     
               <li><a href="#">Store</a></li>
               <li><a href="#">Mac</a></li>
               <li><a href="#">iPod</a></li>
               <li><a href="#">iPhone</a></li>
               <li><a href="#">iPad</a></li>
               <li><a href="#">iTunes</a></li>
               <li><a href="#">Support</a></li>
-              <li><a href="#">Support</a></li>
-              <li><input type="text" name="sear-box"><i class="fa fa-search"></i></li>
+              <li class="last-item"><input type="text" name="sear-box"><i class="fa fa-search"></i></li>
            </ul>
         </nav>
     </header>
     <main>
-
-
+    <section class="call-to-action"> 
+        <div class="page-highlight">
+            <h2>Everyone has something to share. See how people are using<br>
+             iPad in their unique ways to add to the world's story.
+            </h2>
```

Comment:
- [ ] In the original page, this is not a text, but an image. You could use the same to make your page look closer to the original one. This change is **optional**

- [ ] This should be a link like the original page. **Also optional**